lib/features/support/hooks.rb

```diff
@@ -201,9 +201,14 @@ def output_received_requests(request_type)
   # future test runs are from a clean slate.
   Maze::Docker.down_all_services
 
-  next if Maze.config.farm == :none
-
-  # Stop the Appium session and server
-  Maze.driver.driver_quit unless Maze.config.appium_session_isolation
-  Maze::AppiumServer.stop if Maze::AppiumServer.running
+  # Specific shutdown when a device farm is configured
+  case Maze.config.farm
+  when :none
+    # Stop the Appium session and server
+    Maze.driver.driver_quit unless Maze.config.appium_session_isolation
+    Maze::AppiumServer.stop if Maze::AppiumServer.running
+  when :local
+    # Acquire and output the logs for the current session
+    Maze::Runner.run_command("log show --predicate '(process == \"#{Maze.config.app}\")' --style syslog --start '#{Maze::Logger.instance.start_time}' > #{Maze.config.app}.log")
+  end
```

Comment:
There are a couple of problems with the logic here.  Previously `farm == :none` would do nothing and `:local` also includes the use of iOS and Android devices held locally, which we won't want to capture system logs for.

`hooks.rb` probably needs a wholesale refactor at some point to abstract away the complications of different test environments, but for the time being I'd be tempted to just revert to what we had before and add the new line guarded by `if Maze.config.os == 'macos'` (mirroring what we do in the `Before` hook).

```suggestion
  next if Maze.config.farm == :none

  # Stop the Appium session and server
  Maze.driver.driver_quit unless Maze.config.appium_session_isolation
  Maze::AppiumServer.stop if Maze::AppiumServer.running

  if Maze.config.os == 'macos'
    # Acquire and output the logs for the current session
    Maze::Runner.run_command("log show --predicate '(process == \"#{Maze.config.app}\")' --style syslog --start '#{Maze::Logger.instance.start_time}' > #{Maze.config.app}.log")
  end
```